### PR TITLE
Stop suppressing non-JSON logs

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/F8uizllI/537-fix-kibana-not-ingesting-email-alert-service-logs

Please see the commits for more details.

## Testing

Screenshot 2020-10-14 at 13 41 13

## Before

Screenshot 2020-10-14 at 13 39 39

## After

Screenshot 2020-10-14 at 13 39 57

### CODE REVIEW

File: modules/filebeat/manifests/prospector.pp [hunk 1]
```diff
@@ -30,12 +30,12 @@
...
-  $json                  = {},
+  $json                  = false,
   $tags                  = [],
   $symlinks              = false,
 ) {
 
-  validate_hash($fields, $multiline, $json)
+  validate_hash($fields, $multiline)
   validate_array($paths, $exclude_files, $include_lines, $exclude_lines, $tags)
   validate_bool($tail_files, $close_renamed, $close_removed, $close_eof, $clean_removed, $symlinks)
```
Comment (kevindew) [inline modules/filebeat/manifests/prospector.pp:40]: For consistency with existing validating should $json be in this validate_bool grouping? (does seem somewhat inconsistent with most of our puppet to be validiating bools though)

**Review by kevindew — APPROVED** [review]
Nice one, this looks a great improvement. 👍

File: modules/filebeat/manifests/prospector.pp (hunk 1 above)
Comment (benthorner) [reply modules/filebeat/manifests/prospector.pp]: Oh yes, I totally missed that! May as well validate, since it's there...

Comment (benthorner) [reply modules/filebeat/manifests/prospector.pp]: ...and it turns out I forgot to change a couple of the (truthy) hashes, so it was useful in this case!

**Pull request merged**